src/backend/expungeservice/app.py

```diff
@@ -15,6 +16,10 @@ def create_app(env_name):
     app = Flask(__name__)
     app.config.from_object(app_config[env_name])
 
+    logHandler = logging.StreamHandler()
```

Comment:
This is probably just something about python style that I'm ignorant of, but why do things switch to camelCase here?

Reply:
Ah. Because I copied a line for the web and didn't think about style. :) I can fix that.